rust/README.md

```diff
@@ -0,0 +1,19 @@
+# Rust development guide
+
+Firezone uses Rust for all data plane components in the product. This directory
+contains all of the low-level networking immplementations related to STUN/TURN,
+and the Rust-based Linux and Windows clients which are contained in this
+directory.
+
+## Developer Setup
+
+Ensure that `rustup` is installed on your system and available in your `PATH`.
+If not, follow the instructions for your platform.
+
+Then, ensure you have the project toolchain installed with:
+
+```sh
+rustup show
+```
+
+Then installed necessary targets depending on your platform and component.
```

Comment:
```suggestion
Install rustup from https://rustup.rs/
Run `cargo --version` to make sure you have Rust installed.
```

Rustup installs the toolchain for the host platform by default.

Reply:
Wait, i meant to make these part of a review. Sorry, Github.

Reply:
Updated